wasmbinding/query_plugin.go

```diff
@@ -138,6 +173,32 @@ func CustomQuerier(qp *QueryPlugin) func(ctx sdk.Context, request json.RawMessag
 	}
 }
 
+// NormalizeReponses normalizes the responses by unmarshalling the response then marshalling them again.
```

Comment:
The comment seems to be outdated. Could you please explain why we need this conversion in the spec?

Reply:
Good catch! Updated the comment with reasons why this has to change